# -JS中try.. catch..的用法

try 测试代码块的错误。

catch 语句处理错误。

throw 创建并跑出错误。
 try
   {
   //在这里运行代码
     抛出错误
   }
 catch(err)
   {
   //在这里处理错误
   }
